src/core/polyhedral/llvm_jit.cc

```diff
@@ -29,17 +31,55 @@
 
 using namespace llvm;
 
+// Parse through ldconfig to find the path of a particular
+// shared library. This is an unfortunate way to have to
+// find it, but I couldn't immediately find something in
+// imported libraries that would resolve this for us.
+std::string find_library_path(std::string library) {
+  std::string command = "ldconfig -p | grep " + library;
+
+  FILE* fpipe = popen(command.c_str(), "r");
```

Comment:
@wsmoses  oh wow that is really painful.. can't we just set some macro at compile time like I've been doing for CUB?
https://github.com/facebookresearch/TensorComprehensions/blob/master/CMakeLists.txt#L113